### ISSUE TITLE
redshift-gtk tray icon error

### ISSUE THREAD

**Issue by eNTi** [jonls/redshift#425]
A few days ago the tray icon from redshift-gtk was replaced with a placeholder. I'm on manjaro using fluxbox.

Command line shows:

```
(redshift-gtk:6445): Gtk-WARNING **: Theme parsing error: gtk.css:4068:12: Expected a string.

(redshift-gtk:6445): Gtk-WARNING **: Theme parsing error: gtk.css:4071:66: Using one color stop with linear-gradient() is deprecated.

(redshift-gtk:6445): Gtk-WARNING **: Theme parsing error: gtk.css:4249:72: Using one color stop with linear-gradient() is deprecated.

(redshift-gtk:6445): Gtk-WARNING **: Theme parsing error: gtk.css:4256:62: Using one color stop with linear-gradient() is deprecated.

(redshift-gtk:6445): Gtk-WARNING **: Theme parsing error: gtk.css:4259:73: Using one color stop with linear-gradient() is deprecated.

(redshift-gtk:6445): Gdk-CRITICAL **: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
```

**Comment by jonls** [comment]
I'm closing this since it has been a while with no activity. Please reopen if this is still an issue.

**Comment by pavanjadhaw** [comment]
I am having same problem on my system running arch + i3wm
`~ master*
❯ redshift-gtk 

(redshift-gtk:6700): Gtk-WARNING **: Theme parsing error: gtk.css:5394:14: 'color' is not a valid color name

** (redshift-gtk:6700): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-Oj2kriNVqv: Connection refused
Traceback (most recent call last):
  File "/usr/bin/redshift-gtk", line 27, in <module>
    run()
  File "/usr/lib/python3.6/site-packages/redshift_gtk/statusicon.py", line 362, in run
    RedshiftStatusIcon(c)
  File "/usr/lib/python3.6/site-packages/redshift_gtk/statusicon.py", line 93, in __init__
    autostart_item.set_active(utils.get_autostart())
  File "/usr/lib/python3.6/site-packages/redshift_gtk/utils.py", line 71, in get_autostart
    dfile, path = open_autostart_file()
  File "/usr/lib/python3.6/site-packages/redshift_gtk/utils.py", line 39, in open_autostart_file
    autostart_dir = xdg.BaseDirectory.save_config_path("autostart")
AttributeError: module 'xdg' has no attribute 'BaseDirectory'
`

**Comment by jonls** [comment]
@pavanjadhaw This is a different error from what was reported in this issue. If you are using a package based on the master branch, please update. I think this issue was already fixed (see #492). If that does not help, please open a separate issue.
